gui_qt/sync_translation_report.py

```
@@ -89,10 +93,16 @@ def summarize_sync_translation_output(
     message = "同步翻译预览已生成；项目脚本尚未修改。请检查 diff 后再确认写回。"
     if files_done:
         message = f"已处理 {files_done} 个文件。{message}"
-    if lines_to_translate > 0 and translated_count < lines_to_translate:
+    translation_is_partial = (
+        lines_to_translate > 0 and translated_count < lines_to_translate
+    )
+    if preview_status_partial:
+        message = f"部分文件未通过写回计划校验，其余安全预览已生成。{message}"
```

Ruff (0.16.0)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)